pkg/api/devops/v1alpha3/pipeline_types.go

```diff
@@ -44,17 +41,17 @@ const (
 
 // PipelineSpec defines the desired state of Pipeline
 type PipelineSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
 	Type                string               `json:"type" description:"type of devops pipeline, in scm or no scm"`
 	Pipeline            *NoScmPipeline       `json:"pipeline,omitempty" description:"no scm pipeline structs"`
 	MultiBranchPipeline *MultiBranchPipeline `json:"multi_branch_pipeline,omitempty" description:"in scm pipeline structs"`
+
+	// Template contains template referent and parameters to be used to render the execution definition.
+	//+optional
+	Template *PipelineTemplate `json:"template,omitempty"`
```

Comment:
If you want to change the CRD, please use a new version.

Reply:
Thank you for your reminder. I will revert those changes.